transport/wseb/src/main/java/org/kaazing/gateway/transport/wseb/WsebAcceptor.java

```diff
@@ -107,7 +111,7 @@
 import org.kaazing.mina.netty.IoSessionIdleTracker;
 import org.kaazing.mina.netty.util.threadlocal.VicariousThreadLocal;
 
-// TODO: will need some sort of session cleanup timeout
+@SuppressWarnings("deprecation")
```

Comment:
I just wanted to get rid of the deprecation compile warnings (of which there are many) so they don't make it hard to see others. In Eclipse the deprecated classes and methods are still clearly visible as underlined. I can't see us ever removing the deprecated classes and methods in this code base.